cherryontech/jupiter-jumpers
# Logging mood

## Description

1) Updates the mood prompts/three wins modal UI

## Related Issue

N/A

## Related Story Card

[SCRUM 50](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-50)
[SCRUM 34](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-34)


## Type of Changes

Styling update

## Acceptance Criteria

- [X] Given: I have logged into UnCLoud 
- [X] When: I am presented with a selection of reflection optios (e.g. Weather, Goal, or Free form)
- [X] Then: the system allows me to easily choose the option that best reflects my current state
- [X] Given I am on the reflection writing screen
- [X] When I choose the "list daily wins" option
- [X] Then I am provided with space to record my accomplishments

## Update Screenshots

### After
![Screenshot 2024-05-25 at 9 39 47 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/c80e905f-719e-4b69-bfe8-68ca1dc81a78)

## Testing Instructions

1. Initiate a new log on a day without a log. Select a mood.
3. Try clicking 'Done' on the first page, and ensure it properly closes the modal and records the mood, and only renders the mood in the summary of that log (no wins or reflections content).
4. Initiate a new log on a day without a log. Select a mood. Press 'Continue'. Confirm you see your selected mood **centered** in the array of mood icons, with the other (unselected) mood icons made more opaque.
5. Confirm you can see two boxes: one to select your three wins, and one to write a reflection.
6. Toggle the three wins box. Write one win, and ensure that the blue circle fills in with darker blue on the left and with the number 1. Repeat for the other wins. 
7. Toggle the box for the reflection. Do not click inside the select box (use the default question). Write a sample response. Click "Add another reflection".
8. Ensure the previous reflection box automatically collapses. Ensure that in this new reflection component that the only available question(s) in the select box are those that have NOT been yet selected.
9. Repeat the "Add another reflection" clicking until there are no more questions left (it might be two for all of them right now). Ensure that the "Add another reflection" button does no render when there aren't any available questions left.
10. Click 'Done' and ensure that the log saves both the three wins and the reflections and shows them in the summary view for that log.
11. Ensure the progress bar fills to 100% before the modal closes.

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
